File: android/app/src/main/java/com/example/android/domain/model/ProPackage.kt
package com.example.android.domain.model

data class ProPackage(
    val id: String,
    val name: String,
    val price: Double,
    val durationDays: Int,
    val isActive: Boolean = true,
    val createdAt: Long = System.currentTimeMillis(),
    val updatedAt: Long = System.currentTimeMillis()
)

File: backend/src/domain/models/ProPackage.js
const mongoose = require('mongoose');

const proPackageSchema = new mongoose.Schema({
  _id: { type: String, required: true },
  name: { type: String, required: true },
  price: { type: Number, required: true },
  duration_days: { type: Number, required: true },
  is_active: { type: Boolean, default: true }
}, { 
  timestamps: { createdAt: 'created_at', updatedAt: 'updated_at' } 
});

module.exports = mongoose.model('ProPackage', proPackageSchema);